packages/ui/src/RangeBar.tsx

```diff
@@ -0,0 +1,132 @@
+import React, {
+  useContext,
+  useState,
+  useEffect,
+  ForwardedRef,
+  forwardRef,
+  HTMLProps,
+} from 'react'
+import styled from 'styled-components'
+
+import { Text } from './Text'
+
+export interface RangeBarProps extends HTMLProps<HTMLInputElement> {
+  /**
+   * Step value of scroll bar
+   */
+  stepvalue?: number
```

Comment:
no need to change `step` to `stepvalue`, that is a native behavior, also we could change the name of this component to `RangeInput` then `step` would make more sense as well.